test/e2e/adminUI/tests/group005Fields/code/uiTestCodeField.js

```diff
@@ -1,46 +1,8 @@
-module.exports = {
-	before: function (browser) {
-		browser
-		browser.app = browser.page.app();
-		browser.signinPage = browser.page.signin();
-		browser.listPage = browser.page.list();
-		browser.initialFormPage = browser.page.initialForm();
-
-		browser.app.navigate();
-		browser.app.waitForElementVisible('@signinScreen');
-
-		browser.signinPage.signin();
-		browser.app.waitForElementVisible('@homeScreen');
-	},
-	after: function (browser) {
-		browser.app.signout();
-		browser.end();
-	},
-	'Code field should be visible in initial modal': function (browser) {
-		browser.app
-			.click('@fieldListsMenu')
-			.waitForElementVisible('@listScreen')
-			.click('@codeListSubmenu')
-			.waitForElementVisible('@listScreen');
-
-		browser.listPage
-			.click('@createFirstItemButton');
+var fieldTests = require('../commonFieldTestUtils.js');
 
-		browser.app
-			.waitForElementVisible('@initialFormScreen');
-
-		browser.initialFormPage.section.form.section.codeList.section.name
-			.verifyUI();
-
-		browser.initialFormPage.section.form.section.codeList.section.fieldA
-			.verifyUI();
-	},
-	// UNDO ANY STATE CHANGES -- THIS TEST SHOULD RUN LAST
-	'restoring test state': function (browser) {
-		browser.initialFormPage.section.form
-			.click('@cancelButton');
-
-		browser.app
-			.waitForElementVisible('@listScreen');
-	},
+module.exports = {
+	before: fieldTests.beforeUI,
```

Comment:
Because I had it in my head that the before functions were different for UI and UX, since using the UI before in the UX tests didn't work when I tried it. But yeah, there's no difference apart from the additional `browser`...